specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/preview/2021-03-01-preview/connectedClusters.json

```diff
@@ -0,0 +1,801 @@
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/OperationList"
+            }
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ListConnectedClusterOperationsExample": {
+            "$ref": "./examples/ListConnectedClusterOperationsExample.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "OperationList": {
+      "description": "The paginated list of connected cluster API operations.",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Operation"
+          },
+          "description": "The list of connected cluster API operations."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The link to fetch the next page of connected cluster API operations."
+        }
+      }
+    },
+    "Operation": {
+      "type": "object",
+      "description": "The Connected cluster API operation",
+      "properties": {
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Operation name: {Microsoft.Kubernetes}/{resource}/{operation}"
+        },
+        "display": {
+          "readOnly": true,
+          "description": "The object that represents the operation.",
+          "properties": {
+            "provider": {
+              "type": "string",
+              "description": "Service provider: Microsoft.connectedClusters"
+            },
+            "resource": {
+              "type": "string",
+              "description": "Connected Cluster Resource on which the operation is performed"
+            },
+            "operation": {
+              "type": "string",
+              "description": "Operation type: Read, write, delete, etc."
+            },
+            "description": {
+              "type": "string",
+              "description": "Description of the operation."
+            }
+          }
+        }
+      }
+    },
+    "ConnectedClusterList": {
+      "description": "The paginated list of connected Clusters",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of connected clusters",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConnectedCluster"
+          }
+        },
+        "nextLink": {
+          "description": "The link to fetch the next page of connected cluster",
+          "type": "string"
+        }
+      }
+    },
+    "ConnectedCluster": {
```

Comment:
>ConnectedCluster [](start = 5, length = 16)

@Krupesh - Do we need a property for rbac?

Reply:
No we don't need. All Authz info will stay on cluster.